### DISCUSSION TITLE
Env variables

### DISCUSSION THREAD

**Discussion by diego-rapoport** [maaslalani/nap#6] [Q&A] [answered]
Hi, loved the project and I'm already using it!!
Just a quick question, where's the data saved when $XDG_DATA_HOME not set?

**Comment by maaslalani** [comment, marked as answer]
Hey @diego-rapoport,

We use https://github.com/adrg/xdg which defines some defaults for when this variable is unset.

It would be:

```~/.local/share``` on unix
```~/Library/Application Support``` on macOS

if the `$XDG_DATA_HOME` is undefined.

**Reply by OlaoluwaM** [reply]
Can this be modified?

**Reply by maaslalani** [reply]
Yes, set `NAP_HOME` to change where the snippets are stored.

**Comment by diego-rapoport** [comment]
Found it!! Thank you very much!!
